QHyper/solvers/advantage/advatnage.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any, Optional
+
+from dimod import BinaryQuadraticModel
+
+from QHyper.solvers.converter import Converter
+from QHyper.problems.base import Problem
+from QHyper.solvers.base import Solver
+
+from dwave.system import DWaveSampler, EmbeddingComposite
+
+class Advantage(Solver):
+    """
+    Class for solving a problem using
+    Advantage 
+    """
+
+    def __init__(self, problem: Problem, time: float) -> None:
+        self.problem: Problem = problem
+        self.time: float = time
+
+    def solve(self, params_inits: dict[str, Any] = {}) -> Any:
+        sampler = DWaveSampler(region="eu-central-1", solver='Advantage_system5.4')
+        Q = self.problem.objective_function.dictionary
+        sampleset = EmbeddingComposite(sampler).sample_qubo(Q)
```

Comment:
You should use converter methods (`create_qubo`) there to get QUBO here and use `weights` from `params_inits`

Reply:
I will use `Converter.create_weight_free_qubo(problem)` in line 23 because in Advantage we don't use weights.
If I'm missing something or don't know something, please let me know

Reply:
But then you will remove constraints from the problem, because `create_weight_free_qubo` create qubo only from objective function. And I'm pretty sure that's not what you want ;)
It would be a different story if advantage would accept constraints in different form (like in cqm), but it's not the case here.